include/powsybl/iidm/Terminal.hpp

```diff
@@ -62,6 +62,8 @@ class Terminal : public MultiVariantObject {
 
     stdcxx::Reference<Connectable> getConnectable();
 
+    virtual const std::string& getConnectionInfo() const = 0;
```

Comment:
To be discussed: this method is uncessary in C++ as it is used in the `notifyUpdate` call in Java, which does not exist in C++.

Reply:
If it used only for notification, we can remove it